index.js

```diff
@@ -1,14 +1,19 @@
-var _ = require('lodash');
+// const _ = require('lodash');
+const { getExperiences } = require('./query-salesforce');
```

Comment:
Looks like hardcoded use of a specific file's implementation, which can't see apos etc.

This should be a method that starts out empty in the npm module, and gets implemented to return a promise at project level.

The query should also be structured to take place as often as changed data might be relevant not only at initial startup (like in our old PoC). I would expect it to be used by a backend route of a custom schema field type for making a multiple selection of experiences.